sharepoint/sharepoint-ps/sharepoint-online/Set-SPOApplication.md

```diff
@@ -64,8 +66,34 @@ Set-SPOApplication -OwningApplicationId 423poi45-jikl-9bnm-b302-1234ghy56789 -Ov
 ```
 This example demonstrates how to enable file sharing within the SharePoint Embedded application for external users. Note that B2B integration must be enabled to allow guest invitations to SharePoint Embedded apps.
 
+### Example 4
+
+```powershell
+Set-SPOApplication -OwningApplicationId 423poi45 -CopilotEmbeddedChatHosts "http://localhost:3000 https://prepspo.spgrid.com https://pemdevtenant.myownsite.com" 
```

Comment:
>prepspo.spgrid.com [](http://example.com/codeflow?start=106&length=18)

let's not use this example, as this is a real internal tenant. use "contoso" or "fabrikam" in your examples